api-server/server/component-passport.js

```diff
@@ -112,7 +116,8 @@ export const createPassportCallbackAuthenticator = (strategy, config) => (
       if (!user || !userInfo) {
         return res.redirect('/signin');
       }
-      const redirect = `${homeLocation}/learn`;
+      const redirect = `${returnTo}`;
+      console.log('REDIRECT', redirect);
```

Comment:
!!!!

Reply:
I respect the all caps console logging, though. It's the only way.

Reply:
You can't miss it.